BlueShift-iOS-SDK/BlueShiftAlertView.m

```diff
@@ -13,8 +13,12 @@ - (UIAlertController *)alertViewWithPushDetailsDictionary:(NSDictionary *)pushDe
     
     NSDictionary *pushAlertDictionary = [pushDetailsDictionary objectForKey:@"aps"];
     NSString *pushCategory = [[pushDetailsDictionary objectForKey:@"aps"] objectForKey:@"category"];
-    NSString *pushMessage = [pushAlertDictionary objectForKey:@"alert"];
-    UIAlertController *blueShiftAlertController = [UIAlertController alertControllerWithTitle:kAlertTitle message:pushMessage preferredStyle:UIAlertControllerStyleAlert];
+    NSString *pushTitle = [[pushAlertDictionary objectForKey:@"alert"] objectForKey:@"title"];
```

Comment:
Could u please keep the keys in the constant classs

Reply:
The get url params were getting generated in random order, so this way all get params will have a fixed order.